include/kframework/foundry.md

```diff
@@ -1145,20 +1147,19 @@ If the production is matched when no prank is active, it will be ignored.
 ```k
     syntax KItem ::= "#endPrank" [klabel(foundry_endPrank)]
  // -------------------------------------------------------
-    rule <k> #endPrank => . ... </k>
-        <prank>
-          <prevCaller> .Account </prevCaller>
-          <prevOrigin> .Account </prevOrigin>
-          <active> false </active>
-          ...
-        </prank>
-
     rule <k> #endPrank => #clearPrank ... </k>
         <caller> _ => CL </caller>
         <origin> _ => OG </origin>
         <prank>
           <prevCaller> CL </prevCaller>
           <prevOrigin> OG </prevOrigin>
+          <active> true </active>
+          ...
+        </prank>
+
+    rule <k> #endPrank => . ... </k>
+        <prank>
+          <active> false </active>
           ...
         </prank>
 ```
```

Comment:
Why would we be ending the prank for `<active> false </active>`? Is this actually a case we expect to be exercised? Do our tests exercise this case?

Reply:
If not, then maybe the first rule is enough on its own?

Reply:
That rule is for the case in which `vm.endPrank()` is invoked when there is no active prank.

Reply:
Is this a case that Foundry allows?

Reply:
Yes, we have tests for this scenario.